extensions/qute/runtime/pom.xml

```diff
@@ -26,6 +26,11 @@
             <groupId>io.quarkus.qute</groupId>
             <artifactId>qute-core</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.quarkus.qute</groupId>
+            <artifactId>qute-debug</artifactId>
```

Comment:
This should be a conditional dependency: https://quarkus.io/guides/conditional-extension-dependencies#dev-mode-only-extension-dependencies

Reply:
I agree, but I had to do that, because of , `DebugQuteEngineObserver.class`

Reply:
fixed